src/test/java/com/ifmo/epampractice/security/SecureStringTest.java

```diff
@@ -0,0 +1,105 @@
+package com.ifmo.epampractice.security;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.charset.StandardCharsets;
+
+public class SecureStringTest {
+
+    @Test
+    public void stringValueTest() {
+        char[] chars = new char[] { 'n', 'i', 'c', 'e' };
+        SecureString str = new SecureString(chars);
+
+        Assert.assertEquals(chars.length, str.getLength());
```

Comment:
Maybe we can use different assert here? Looks like we can change all this logic(even following loop) with only one assert.

Reply:
Not sure what exactly could be changed here.
If we had char array instead of SecureString, we could use assertArrayEquals().